src/commands/upload_file.ts

```diff
@@ -156,11 +160,13 @@ new CLICommand({
 			const wallet = parameters.getRequiredParameterValue(WalletFileParameter, readJWKFile);
 
 			const conflictResolution = parameters.getFileNameConflictResolution();
+			const shouldBundle = !!parameters.getParameterValue(NoBundleParameter);
```

Comment:
~Is it `shouldBundle` or `shouldNotBundle`?~
- ~if `--no-bundle`, then `parameters.getParameterValue(NoBundleParameter)` is true~
- ~`!!true` is true~

**EDIT**
This is fine because the parameter declarations specify the `bundle` parameter name, but `--no-bundle` is the alias so the negation is already applied.
Maybe the confusing thing here is the name of the parameter. It should be `ShouldBundleParameter` or `BundleParameter`